server/plugin.go

```diff
@@ -35,6 +35,7 @@ const (
 	// Move these two to the plugin settings if admins need to adjust them.
 	WebhookMaxProcsPerServer = 20
 	WebhookBufferSize        = 10000
+	PluginRepo               = "https://github.com/mattermost/mattermost-plugin-jira"
```

Comment:
2/5 `s/PluginRepo/GitHubRepositoryURL`

Reply:
Hello @lev , thanks for the review,
Do you mean something like this

`const PluginRepo := "mattermost-plugin-jira"`
`const GitHubRepositoryURL := "https://github.com/mattermost/"`
`Repo := fmt.Sprintf("%s%s", PluginRepo, GitHubRepositoryURL)`

Reply:
Nope, just asked to rename the const, but it's ok as is